src/helpers/userHelper.js

```diff
@@ -0,0 +1,43 @@
+import { google } from 'googleapis';
+import dotenv from 'dotenv';
```

Comment:
Instead of importing the entire module why not `import { config } from dotenv`
Then call it as `config`

Reply:
Noted